### DISCUSSION TITLE
Is there any roadmap about Mantine V5?

### DISCUSSION THREAD

**Discussion by hexwit** [mantinedev/mantine#1807] [Q&A] [answered]
Hi! 

I am curious is there any topic about v5 roadmap? What would be the difference between 4 and 5?
Key concepts etc.

Thanks.

**Comment by rtivital** [comment, marked as answer]
You can view 5.0 changelog here – https://v5.mantine.dev/changelog/5-0-0/
We do not provide any additional information

**Reply by hexwit** [reply]
If no roadmap, then I would assume it is technical version. Some kind of "refactoring". 
We probably will have the same technologies, but component structure and functionality will change. 

Then question is: should I start new project with v4 or v5? How easy migration would be?

**Reply by rtivital** [reply]
v5 will introduce additional breaking changes sometime later. If you do not want that, choose v4, otherwise use v5.
